test/lib/CustomRulePathManager.test.ts

```diff
@@ -85,15 +85,21 @@ describe('CustomRulePathManager tests', () => {
 
     describe('Adding new Rule Path entries', () => {
```

Comment:
Would be nice to have another test that verifies that `invalidFilePath`error is thrown when `stats` can't find the file.